extension.ts

```diff
@@ -310,15 +310,8 @@ export async function activate(context: vscode.ExtensionContext) {
     }
   });
 
-  registerCommand(context, 'toggleVim', async () => {
-    Globals.active = !Globals.active;
-    if (Globals.active) {
-      await vscode.commands.executeCommand('setContext', 'vim.active', Globals.active);
-      compositionState = new CompositionState();
-      modeHandlerToEditorIdentity = {};
-      let mh = await getAndUpdateModeHandler();
-      mh.updateView(mh.vimState, { drawSelection: false, revealRange: false });
-    } else {
+  async function toggleDisabledMode() {
```

Comment:
Hmm, I'm trying to think of a better name as it's not actually toggling the mode (ie. if you call this twice, it doesn't actually toggle and go from disable->enable). 

Maybe this signature:

`toggleExtension(bool isDisabled)`